### PR TITLE
Correcting sorting in translations view + some.

### PR DESCRIPTION
**Pull request by infograf768**
SearchTools implemented in Translations view and corrected sorting ASC and DESC.
screen shot 2016-03-23 at 09 51 43

Also added missing string `COM_LOCALISE_REVISED="Revised"`
Also corrected Notice when `$develop_file_path` has not yet been created (i.e. when using com_localise the first time without testing/implementing Github compare).

### CODE REVIEW

**Comment by infograf768** [comment]
@Valc 

Need tests here to be able to release a new version.

**Comment by Valc** [comment]
Oks, i go to work now but i test it this afternoon.
Regards.

**Comment by Valc** [comment]
Tested and working fine here :+1:

**Comment by infograf768** [comment]
thanks

File: component/admin/models/translations.php [hunk 1]
```diff
@@ -876,18 +897,10 @@ private function getTranslations()
...
-				$direction = 1;
-			}
-
-			JArrayHelper::sortObjects(
-				$this->translations,
-				$this->getState('list.ordering', 'name'),
-				$direction
-			);
+			// Process ordering.
+			$listOrder = $this->getState('list.ordering', 'name');
```
Comment (andrepereiradasilva) [inline component/admin/models/translations.php:901]: 'filename'  like in the xml?

**Pull request merged**